ci-operator/step-registry/stackrox/collector/update-support-packages/stackrox-collector-update-support-packages-workflow.yaml

```
@@ -0,0 +1,26 @@
+workflow:
+  as: stackrox-collector-update-support-packages
+  steps:
+    pre:
+      - ref: stackrox-collector-update-support-packages-setup
+    test:
+      - as: stackrox-collector-update-support-packages
+        from: src
+        commands: .openshift-ci/jobs/update-support-packages/run.sh
+        credentials:
+          - mount_path: /tmp/secret/stackrox-collector-e2e-tests
+            name: stackrox-collector-e2e-tests
+            namespace: test-credentials
+        resources:
+          requests:
+            cpu: 1000m
+            memory: 500Mi
+        timeout: 4h0m0s
+        grace_period: 5m0s
+        env:
+        - name: VM_TYPE
+        - name: IMAGE_FAMILY
```

Since this step is used with the same parameters, we might want to set them as the `default` here and simplify `stackrox-collector-master.yaml` a little more: https://docs.ci.openshift.org/docs/architecture/step-registry/#declaring-step-parameters
```suggestion
    env:
    - VM_TYPE: ubuntu-os
    - IMAGE_FAMILY: ubuntu-2204-lts
```

---

Done